pandas/src/generate_code.py

```diff
@@ -1,3 +1,10 @@
+"""This file generates `generated.pyx` which is then included in `../algos.pyx`
+during building.  To regenerate `generated.pyx`, just run:
```

Comment:
this actually has to be run from ``pandas/src`` (so that ``generated.pyx`` is in the correct place), so either we need to fix this (e.g. the output path should be relative to the pandas tree), or provide this in the instructions.